### PR TITLE
Check for dao.hypha

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**

### CODE REVIEW

File: lib/core/network/models/transaction_model.dart [hunk 1]
```diff
@@ -32,6 +32,10 @@ sealed class TransactionModel extends Equatable {
     final actor = act['authorization'].first['actor'];
     final transactionId = json['trx_id'];
 
+    if(account != 'dao.hypha') {
```
Comment (gguijarro-c-chwy) [inline lib/core/network/models/transaction_model.dart:35]: husd.hypha but its must be dao.hypha

This check fails.

What should we do @n13

Comment (gguijarro-c-chwy) [reply lib/core/network/models/transaction_model.dart:35]: there is also voice.hypha

File: lib/core/network/models/transaction_model.dart [hunk 2]
```diff
@@ -32,6 +34,13 @@ sealed class TransactionModel extends Equatable {
     final actor = act['authorization'].first['actor'];
     final transactionId = json['trx_id'];
 
+    // if (account != 'dao.hypha') {
```
Comment (gguijarro-c-chwy) [inline lib/core/network/models/transaction_model.dart:37]: this doestn work

there are many accounts, not only dao.hypha

**Comment by n13** [comment]
FIFY ;)

**Comment by n13** [comment]
Fixed it - see checkin

File: lib/core/network/models/transaction_model.dart (hunk 1 above)
Comment (n13) [reply lib/core/network/models/transaction_model.dart]: Fixed

**Review by n13 — APPROVED** [review]

**Pull request merged**